init_suite.py

```diff
@@ -54,22 +57,27 @@ def process_override(dir_name: str) -> List:
     # At this point, the entry/item is a directory. Inside this folder, we
     # support only three files
     override_data = dict()
-    test_data = dict()
+    #test_data = dict()
+    test_data = dict({'tests':list()})
 
     log.debug(f"Processing overrides of {dir_name}")
 
     files = glob(os.path.join(dir_name, "*"))
+    files.sort()
```

Comment:
The idea is to use file-system structure so that we maintain the sequence/precedence. Any particular reason why this was introduced.

Reply:
If the directory contains many files then we sort it based on the filenames (the filenames must be prefixed with number), so that we can use the 'index' key in overrides test to override the particular test as intended by the user